### PR TITLE
[4.4] Cypress Update README.md

### PR DESCRIPTION
**Pull request by brianteeman**
small grammar/english corrections

### CODE REVIEW

File: tests/System/README.md [hunk 1]
```diff
@@ -11,21 +11,21 @@ A couple of steps are needed before the CMS system tests can be executed on the
    2. `npm ci`
 3. Copy the cypress.config.dist.js to cypress.config.js in the root of the joomla folder
 4. Adapt the env variables in the file cypress.config.js, they need to point to a database server where joomla can be installed
```
Comment (sandewt) [inline tests/System/README.md:13]: It took me some time to figure out that I needed to change the base url. Would you like to add some text?

`Adjust the baseUrl and adapt the env ...`

Comment (laoneo) [reply tests/System/README.md:13]: Can you do that in a new pr @sandewt ?

**Comment by laoneo** [comment]
Thanks!

File: tests/System/README.md (hunk 1 above)
Comment (sandewt) [reply tests/System/README.md:13]: I'll do that.

Comment (sandewt) [reply tests/System/README.md:13]: Done

**Pull request merged**